Resources/Locale/en-US/deltav/reagents/pharmacymsg.ftl

```diff
@@ -0,0 +1,42 @@
+reagent-effect-painkiller-light1 = You feel a tiny bit better.
+reagent-effect-painkiller-light2 = The pain is manageable. For now.
+reagent-effect-painkiller-light3 = It will be okay, the pain is slowly fading.
+
+reagent-effect-painkiller-head-mild1 = Your head aches less.
+reagent-effect-painkiller-head-mild2 = You don't feel woozy anymore.
+
+reagent-effect-painkiller-head-strong1 = Your head feels numb.
+reagent-effect-painkiller-head-strong2 = You feel like you can think properly.
+reagent-effect-painkiller-head-strong3 = You can finally focus.
+reagent-effect-painkiller-head-strong4 = You feel extremely relieved.
+
+reagent-effect-painkiller-head-fade = You feel like your head is about to explode.
+
+
+reagent-effect-asparatine-overdose = Your tummy aches.
+reagent-effect-asparatine-overdose2 = It feels like you're getting stabbed in the guts.
+reagent-effect-asparatine-overdose3 = It is digging inside you.
+
+reagent-effect-asparatine-naguuprofen-mix1 = The Asparatine and Naguuprofen weren't a good idea...
```

Comment:
If this was not copy-pasted directly from CD, we might avoid ick-ock. The character who took the medicine does not necessarily know that they have taken these drugs.